### PR TITLE
Implement CreateWorkout screen

### PR DESCRIPTION
**Pull request by klymenkoo**
No back-end logic.
create workout

### CODE REVIEW

File: components/ExerciseCard.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import { View, StyleSheet, Text, TextInput } from "react-native";
+import { Button, Icon } from "react-native-elements";
+import WorkoutCard from "../components/WorkoutCard";
+import { FontStyles } from "../styles/global";
+import SetDetailsView from "../screens/SetDetailsView";
+import { isTSExpressionWithTypeArguments } from "@babel/types";
```
Comment (AlexeiGitH) [inline components/ExerciseCard.js]: What is this used for?

File: components/ExerciseCard.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    };
+
+    this.addSet = this.addSet.bind(this);
+    this.changeName = this.changeName.bind(this);
+    this.changeDesc = this.changeDesc.bind(this);
+  }
+
+  addSet() {
+      let set ={
+        id: 0,
```
Comment (AlexeiGitH) [inline components/ExerciseCard.js]: Not sure if we should id's manually or firebase generates something unique. 
Can fix it later I think

File: screens/WorkoutDetails.js [hunk 3]
```diff
@@ -67,9 +67,9 @@ class WorkoutDetailsScreen extends React.Component {
                         </View>
                         <View style={styles.exercises}>
                             {
-                                this.workout.exercises.map((exercise, ei) => {
+                                this.workout.exercises.map((exercise, id) => {
```
Comment (AlexeiGitH) [inline screens/WorkoutDetails.js]: Just to make sure, second parameter 'ei' is just an index (starting from 0 to N), it's not an ID of the element. Don't know if you need an id or just a number in a sequence.

File: components/ExerciseCard.js (hunk 1 above)
Comment (klymenkoo) [reply components/ExerciseCard.js]: hmmm, weird. Not sure where it came from. Probably, I accidentally copied it. Good catch 👍

**Pull request merged**